pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/operations/configuration/index.md

```diff
@@ -66,3 +66,56 @@ In order for {{ model.techShortName }} to function correctly, you must perform s
   </tr>
 
 </table>
+
+## Using Volume Profiles
+
+Volume profiles are used to classify volumes. For example, users can group SSDs into a “fast” profile and group HDDs into a “slow” profile. 
+
+<p class="message--note"><strong>NOTE: </strong>Volume profiles are immutable and therefore cannot contain references to specific devices, nodes or other ephemeral identifiers.</p> 
+
+DC/OS Storage Service (DSS) is a service that manages volumes, volume profiles, volume providers, and storage devices in a DC/OS cluster.
+
+If you want to deploy Hdfs with DSS, please follow [this tutorial](/mesosphere/dcos/services/storage/1.0.0/tutorials/cassandra-dss-volumes/) which illustrates, how to deploy Cassandra with DSS.
```

Comment:
```suggestion
```
Let's delete this reference to avoid confusing users

Reply:
Done